### ISSUE TITLE
saveFile returns file path on iOS, but only directory path on Android

### ISSUE THREAD

**Issue by WieFel** [incrediblezayed/file_saver#16]
I am using the following code:
```dart
  Future<String> saveFile() async {
    String content = "hello";
    Uint8List bytes = Uint8List.fromList(utf8.encode(content));
    final path = await FileSaver.instance.saveFile("testfile", bytes, "txt");
    return path;
  }
```

On iOS, this function correctly returns the whole file path:
```
/var/mobile/Containers/Data/Application/8532ACC4-83A1-4557-A6DF-9F5EF7F0F724/Documents/testfile.txt
```

But on Android, it only returns the enclosing directory path without the filename:
```
/storage/emulated/0/Android/data/com.example.pdf_flutter/files
```

Is this behaviour expected?

**Comment by incrediblezayed** [comment]
Not intended, but expected, I'll fix that as soon as possible.

**Comment by WieFel** [comment]
@incrediblezayed thanks! Would be cool!
Because currently I have to check if I'm on Android and then add the filename manually...

**Comment by incrediblezayed** [comment]
> @incrediblezayed thanks! Would be cool!
> Because currently I have to check if I'm on Android and then add the filename manually...

Sorry for the inconvenience, you can even contribute in the plugin, since I've took on multiple project I'm not able to manage time.

**Comment by incrediblezayed** [comment]
Hello, Thank You for pointing out the issue, it's fixed now, use v 0.1.0
